Quadriga/src/main/webapp/WEB-INF/views/workbench/public/publicpagestatistics.jsp

```diff
@@ -0,0 +1,66 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
+<%@taglib uri="http://www.springframework.org/tags/form" prefix="form"%>
+
+<script>
+	$(function() {
+		$("input[type=submit]").button().click(function(event) {
+		});
+
+		$("input[type=button]").button().click(function(event) {
+		});
+	});
+
+	$(function() {
+		$("input[name='selectAll']").button().click(function() {
+
+			$("form input:checkbox").prop("checked", true);
+			event.preventDefault();
+			return;
+		});
+
+		$("input[name='deselectAll']").button().click(function() {
+
+			$("form input:checkbox").prop("checked", false);
+			event.preventDefault();
+			return;
+
+		});
+
+	});
+</script>
+<input type="button" value="Select All" name="selectAll">
```

Comment:
How does the user know what they have to do on this page? Some more info what the user can do here is needed.